### DISCUSSION TITLE
Tiempo Ejecución

### DISCUSSION THREAD

**Discussion by bozzimuc** [IIC2133-PUC/2024-2#65] [Tarea 0] [answered]
Buenas tardes. Cuando corro el código con time, ¿cual de los 3 tiempos que salen (real, user, o sys) es el que se compara a los 5 segundos? En otra issue se recomendaba usar codespaces, pero no logré entender como funciona. Muchas gracias

**Comment by tomastrivino** [comment, marked as answer]
Real

Éxito
